api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentDragging/onDragEnd.md

```diff
@@ -19,25 +19,25 @@ The UI component's instance.
 #include common-ref-eventparam
 
 ##### field(e.fromComponent): dxSortable | dxDraggable
-The instance of the appointment's source UI component.
+The instance of the appointment's source UI component. Works only for dragging between components.
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/appointmentDragging/data.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/appointmentDragging/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/appointmentDragging/data.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/appointmentDragging/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.itemData): any
-The dragged appointment's data.
+The appointment's data before dragging.
 
 ##### field(e.itemElement): DxElement
 #include draganddrop-ref-itemelementparam
 
 ##### field(e.toComponent): dxSortable | dxDraggable
-The instance of the appointment's target UI component.
+The instance of the appointment's target UI component. Works only for dragging between components.
 
 ##### field(e.toData): any
-Custom [data](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/appointmentDragging/data.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/appointmentDragging/#data') associated with the target UI component.
+Custom [data](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/appointmentDragging/data.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/appointmentDragging/#data') associated with the target UI component. Works only for dragging between components.
 
 ##### field(e.toItemData): any
-<!-- Description goes here -->
+The appointment's data after dragging.
```

Comment:
In the "onDragEnd" handler, the user can cancel the drag-n-drop event by setting the ```e.cance = true;```

So, maybe it will be better to write here something like:
"The alleged appointment's data after dragging completes."
or
"The appointment's data after dragging completes".

Reply:
I think the "The appointment's data after dragging completes" expression means the same as "The appointment's data after dragging."